tests/utils.js

```diff
@@ -107,6 +107,20 @@ describe('Utils', () => {
         });
     });
 
+    describe('assertSaneResponse', () => {
+        it('should throw on falsy response', () => {
```

Comment:
Can we keep the same naming pattern as in other test cases, namely: `it should thrown an error ...` ?